app/Http/Controllers/EInvoiceController.php

```diff
@@ -120,14 +121,17 @@ public function quota(ShowQuotaRequest $request): \Illuminate\Http\Response
                 'account_key' => $company->account->key,
             ]);
 
-        if ($response->successful()) {
-            return response($response->body());
+
+        if ($response->status() == 422) {
+            return response()->json(['message' => $response->json('message')], 422);
         }
 
         if ($response->getStatusCode() === 400) {
-            return response($response->body(), 400);
+            return response()->json(['message' => $response->json('message')], 400);
         }
 
-        return response()->noContent(500);
+        return response()->json([
```

Comment:
->json() inside ->json() does that double encode?

Reply:
No, I believe $response->json is to get json key value.